crates/astria-sequencer-relayer/tests/blackbox/main.rs

```diff
@@ -61,7 +64,9 @@ async fn report_degraded_if_block_fetch_fails() {
         ),
     )
     .await
-    .expect("requesting abci info and block must have occured");
+    .expect("requesting abci info and block must have occured")
+    .1
```

Comment:
Yeah, that's weird but OK right now. We should fix this (I added a comment on `wait_until_satisifed()`)